src/Spice86.Core/Emulator/OperatingSystem/VirtualDeviceBase.cs

```diff
@@ -0,0 +1,15 @@
+namespace Spice86.Core.Emulator.OperatingSystem;
+
+public abstract class VirtualDeviceBase : IVirtualDevice {
+    protected VirtualDeviceBase(DeviceAttributes attributes, ushort strategy = 0, ushort interrupt = 0) {
```

Comment:
Can the constructor be documented in a XML summary ?
Especially the strategy param

Reply:
Done